devices/xiaomi/xiaomi_rtcgqq11lm_presence_sensor.json

```diff
@@ -46,6 +46,19 @@
         },
         {
           "name": "attr/swversion",
+          "parse": {
```

Comment:
Isn't this already reported through the Xiaomi special attribute?

Reply:
Thanks for the tip, I'll try it again.

Reply:
It seems to work.